Source/Basic Shapes/SvgVisualElement.cs

```diff
@@ -279,41 +279,87 @@ protected internal virtual bool RenderStroke(ISvgRenderer renderer)
                         {
                             using (var pen = new Pen(brush, strokeWidth))
                             {
-                                if (this.StrokeDashArray != null && this.StrokeDashArray.Count > 0)
+                                if (StrokeDashArray != null && StrokeDashArray.Count > 0)
                                 {
-                                    if (this.StrokeDashArray.Count % 2 != 0)
+                                    strokeWidth = strokeWidth <= 0 ? 1 : strokeWidth;

```

Comment:
What this condition is or will be true in case ?

Reply:
This has been used below twice - I just factored out for better readability of the code where it is used.
It seems to make sure that stroke width is set to 1 if StrokeWidth is invalid.

Reply:
It is necessary to confirm in line 248. . .
```cs
StrokeWidth > 0
```